app/assets/javascripts/config.js.coffee.erb

```diff
@@ -13,6 +13,9 @@
   match = /cmr_env=(\w+)/.exec(query)
   cmr_env = if match then match[1] else '<%= Rails.configuration.cmr_env %>'
 
+  
+  reverb_url = "<%= ENV['reverb_url']%>"
+
   if cmr_env == 'uat'
     browse_scaler_url = "<%= ENV['browse_scaler_url_uat'] ||  Rails.configuration.services['earthdata']['uat']['browse_scaler_root'] %>"
```

Comment:
If you start rails server like this: ```browse_scaler_url_uat=https://uat.url rails s```
The line above tries to set ```browse_scaler_url``` using the ENV set in command line being ```https://uat.url```.

If you start the rails server ```rails s```, ```ENV['browse_scaler_url_uat']``` will be nil. So the code continues to take a look at the second part ```Rails.configuration.services['earthdata']['uat']['browse_scaler_root']``` to try to get the default value from there.

Reply:
You need to ```export``` the variable too. See below in "exports =" lines.